rascal-textmate-core/src/main/rascal/lang/textmate/conversiontests/PicoWithCategories.rsc

```diff
@@ -42,10 +42,22 @@ syntax Expression
            | min: Expression lhs "-" Expression rhs)
     ;
            
-lexical Id = [a-z][a-z0-9]* !>> [a-z0-9];
+lexical Id = ([a-z][a-z0-9]*) !>> [a-z0-9] \ Keyword;
```

Comment:
Added this condition (diverging slightly from `lang::pico::\syntax::Main`) to make the highlighting a bit nicer)